certification/engine/engine.go

```diff
@@ -98,3 +113,25 @@ func AllChecks() []string {
 	}
 	return all
 }
+
+func OperatorPolicy() []string {
```

Comment:
I was originally going to ask to replace the word Policy with Check, BUT - do we want to leave "Policy" as the term here, under the previously discussed concept that our _policy_ is a collection of checks?

Reply:
Exactly.  Policy = a collection of checks.